src/scripts/push_coral_to_hub.py

```diff
@@ -220,43 +255,69 @@ def get_iteration_name(start_time: str | datetime) -> str:
     )
     train_recordings_df = train_recordings_df.merge(
         speaker_metadata.rename(columns=lambda x: f"{x}_1"),
+        on="speaker_id_1",
         how="left",
     )
     train_recordings_df = train_recordings_df.merge(
         speaker_metadata.rename(columns=lambda x: f"{x}_2"),
+        on="speaker_id_2",
         how="left",
     )
 
     # Pick a validation set from the training set, by selecting a single recording
-    # from each speaker. We do this by selecting the last recording from each speaker
-    # in the training set, as the first recordings are conversations.
+    # from each speaker. If make sure we do not select more than 3 conversation
+    # recordings, as these are typically longer and might skew the validation set.
```

Comment:
Avoid hardcoding arguments into comments, it results in errors like here.

```suggestion
    # from each speaker. If make sure we do not select too many conversation
    # recordings, as these are typically longer and might skew the validation set.
```